src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -53,7 +53,6 @@ export function SavedObjectProvider(esAdmin, kbnIndex, Promise, Private, Notifie
 
     // type name for this object, used as the ES-type
     const esType = config.type;
-    this.index = kbnIndex;
```

Comment:
Why did you remove this?  Are you sure this doesn't have trickle down effects elsewhere, on xpack or something?